politeiad/backendv2/tstorebe/tstorebe.go

```diff
@@ -1017,21 +1018,79 @@ func (t *tstoreBackend) Fsck() error {
 		return err
 	}
 
-	// Rebuild the inventory cache. This is a multi-step process:
-	//
-	// 1. Sort the tokens into two groups; unvetted and vetted. Each
-	//    group is additionally sorted by the timestamp of their most
-	//    recent status change.
-	//
-	// 2. Add the vetted tokens to the inventory cache. They MUST be
-	//    added starting with the oldest timestamp. The vetted token
-	//    is first added to the unvetted cache then moved to the vetted
-	//    cache. This is a limitation of the inventory API and mimicks
-	//    how records are added and updated when using the politeiad
-	//    API.
-	//
-	// 3. Add the unvetted tokens to the inventory cache. They MUST be
-	//    added starting with the oldest timestamp.
+	// Get the partial record for all tokens. This also guarantees that all
```

Comment:
There should be some type of log statement about rebuilding the inventory cache.

Reply:
It would be nice if the number of records in the inventory is in the log statement too.

Reply:
You can use the length of the provided tokens to determine how many records are being added to the inventory.

Reply:
Yep it's done, had to step away for a few but will push once im back 👍

Reply:
pushed @lukebp